### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by safafa**
Responsive web design Milestone1:  
HTML Skeleton contains the navbar, highlights, latest funds, and the footer.
CSS: Using the media queries the webpage responds to screen size. It adapts to mobile less than 768px, Tablet less than 1024px, and large screens.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+
+https://user-images.githubusercontent.com/65791349/113003573-23a1b980-9173-11eb-8a31-7517f6a1b655.png
+
+# Built With
+
+HTML & CSS
+
+# Live Demo
+
+https://1v4n4.github.io/Responsive-web-page/
```
Comment (christianotieno) [inline README.md]: ## Improve the `README`
- [ ] We currently do need to have the name of the authors listed in the `README` and a screenshot to have a descriptive `README`. The format can also be improved further. Kindly use this [template](https://github.com/microverseinc/readme-template) as an inspiration.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
```
Comment (christianotieno) [inline index.html:1]: ## Improve the general layout
- [x] We currently do have all the sections of the page covering the entire view width of the page. This shouldn't be so. We do need to, however, have only the header, the three main articles and the footer covering the full width of the page. the `Latest funding` section ought to be centred properly. Kindly fix this.

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://1v4n4.github.io/Responsive-web-page/)|
|:--:|:--:|
|tnw|tnw-clone|

File: index.html [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+        <title>Safa's and Ivana's responsive page</title>
+    </head>
+
+    <body>
+                <header>
+                    <nav id="header-container">
+                        <div id="small-top-line">
+                            <img class="small-icon" src="img/tnw.png" alt="tnw-logo">
```
Comment (christianotieno) [inline index.html]: ## Code implementation details
- [x] Ensure the logo has a link (a tag).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">SHIFT</a>
+                                    </li>
+                                    <li class="white-top-left inline">
+                                        <a class="white-link no-underline" href="#">NEURAL</a>
+                                    </li>
+                                </ul>
+                            </div>
+                            <div class="right-nav-div" id="white-right-nav-div">
+                                <img src="img/magnifying_glass30x30.png" alt="magnifier-icon">
```
Comment (christianotieno) [inline index.html]: ## Code implementation details
- [x] Ensure the search button(icon) is also a link.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+                        <div class="mb2">
+                            <p>Food delivery startup Wolt raises $530M after tripling revenues on SiliconANGLE</p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+        </main>
+        <!--Footer-->
+        <footer class="cflex">
```
Comment (christianotieno) [inline index.html:214]: ## Improve the footer
- [x] Be mindful of how the social icons have been styled. 
- [x] Some images are also not loading in this section. Do you mind fixing them,?

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://1v4n4.github.io/Responsive-web-page/)|
|:--:|:--:|
|footer|footer-clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 

Hello @safafa & @1v4n4 👋🏾 

Congratulations on submitting your `Building with Responsive Design - Milestone 1` project. 
The code is clean and well maintained. Before we proceed, we need to make sure nothing is messed up. Let's observe the following changes and improve on the highlighted areas below 👇🏾 

## Changes
- [ ] Improve the general layout on a large screen.
- [ ] Improve the README
- [ ] Improve the footer

### Happy coding 💻 
#### Christian Otieno (CR)

File: css/index.css [hunk 6]
```diff
@@ -0,0 +1,514 @@
...
+  }
+
+  .fund > img {
+    width: 100%;
+  }
+
+  .socials {
+    margin-bottom: 10px;
+    width: 10%;
+  }
```
Comment (Berabjesus) [inline css/index.css]: ## Improve the footer
The problem with the footer links seems to originate here. On a screen size greater than 1024 the `.socials` class defaults to 10% of the screen size, take the appropriate measure to allow the footer links to be displayed and spaced correctly.

File: css/index.css [hunk 7]
```diff
@@ -0,0 +1,514 @@
...
+  font-size: 5px;
+}
+
+.bicon {
+  background-color: rgba(0, 0, 0, 0.363);
+  border-color: rgba(0, 0, 0, 0.363);
+  border: none;
+  border-radius: 50%;
+  width: 10%;
+}
```
Comment (Berabjesus) [inline css/index.css]: - [x] [OPTIONAL] No need to use `border-color: rgba(0, 0, 0, 0.363);` if you you are going to use `border: none;` on the next line.
- [x]  [OPTIONAL]  for `border-radius: 50%` the height and width should be equal for a proper circle.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+            <div class="upperf cflex acenter jcenter">
+                <div class="socials flex sjaround">
+                    <button type="submit" class="bicon"><i class="fa fa-facebook-square"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-youtube"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-twitter"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-instagram"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-pinterest"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-facebook"></i></button>
+                    <button type="submit" class="bicon"><i class="fa fa-envelope"></i></button>
+                </div>
```
Comment (Berabjesus) [inline index.html:224]: ## Improve the footer styles:- the social media icons are still not styled properly.

- [x] Give the appropriate width and height for the social media links so the footer links match the [original website](https://perma.cc/M5ZV-Q2D6)
- [x] match the appropriate background color of the website for these links.
- [x] Links should give an impression that they can be clicked. As you can see in the [Source Page](https://perma.cc/M5ZV-Q2D6) the links change the cursor type when a user hovers over them.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
## Status: CHANGES REQUIRED  :warning: :construction: 
Hi @safafa and @1v4n4. Great progress! Awesome README :100: ! Good job with the changes! :tada: We are almost there. We, however, do have a few changes to carry out before we proceed (see review comment).

**For this project, we have to make sure nothing is messed up, especially in the header and footer. Per the previous review, you have one last thing to work on** 
## changes
- [ ] Improving the footer. 

**Please take a look at the comments below and work on them accordingly.** 
### ❗ As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second-opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).  

**When you have made the specified changes, please submit another code review. Thank you!** 
If you have any questions, let me know by leaving a comment here and mentioning me. 
I will be happy to help
HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me) | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @safafa @1v4n4, good job on your TNW's clone🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 
Well done team

[ OPTIONAL ]

- You missed the logo in the 768px breakpoint, add it to the header
- Improve the layout of the content in the last section (latest funding round), use padding and margin to fix it.

Now you can merge this branch 🚀 

You can contact me on slack if you have any questions, just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

**Pull request merged**